lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -541,13 +574,52 @@ def __get_template_with_version(self):
 
         return template
 
+    def __get_storage_domain_and_all_template_disks(self, template):
+
+        if self._module.params.get('template') is None:
+            return None
+
+        if self.param('storage_domain') is None:
+            return None
+
+        storage_domain = self.param('storage_domain')
```

Comment:
I think it's not worth to create the temp var here, maybe just pass the `self.param('storage_domain')`
directly into the `get_id_by_name` ?